### DISCUSSION TITLE
Functional Recommendations

### DISCUSSION THREAD

**Discussion by Phrixus2023** [lllyasviel/Fooocus#771] [Ideas]
Thank you very much to the developers for creating this tool. I have a small suggestion: to support translation functionality in the prompt input box, using free translation interfaces or the like. As an early transition for GPT, this could better promote internationalization and popularization of fooocus.

**Comment by xhoxye** [comment]
a suggestions to try: 
 browser extensions, 
https://immersivetranslate.com/docs/input/
https://github.com/immersive-translate/immersive-translate/releases
